projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -608,6 +608,112 @@ function wpcom_register_launchpad_task_list( $task_list ) {
 	return wpcom_launchpad_checklists()->register_task_list( $task_list );
 }
 
+/**
+ * Retrieves the required tasks from a given set of tasks.
+ *
+ * This function filters the provided tasks array and returns only the tasks that are marked as required, based on the specified
+ * array of required task IDs.
+ *
+ * @param array $required_task_ids An array of task IDs that are considered required.
+ * @param array $tasks An array of tasks to filter.
+ * @return array An array containing only the required tasks from the provided task list.
+ */
+function wpcom_launchpad_get_required_tasks( $required_task_ids, $tasks ) {
+	return array_filter(
+		$tasks,
+		function ( $task ) use ( $required_task_ids ) {
+			return in_array( $task['id'], $required_task_ids, true );
+		}
+	);
+}
+
+/**
+ * Get all tasks that are marked as launch tasks.
+ *
+ * @param array $tasks Array of tasks.
+ * @return array Array of launch tasks.
+ */
+function wpcom_launchpad_get_launch_tasks( $tasks ) {
+	return array_filter(
+		$tasks,
+		function ( $task ) {
+			return $task['isLaunchTask'];
```

Comment:
Just to be extra safe, it may be worth checking if the field is set. Also, `??` was only added in PHP 7, hence the use of `isset()`.
```suggestion
			if ( isset( $task['isLaunchTask'] ) ) {
				return $task['isLaunchTask'];
			}
			return false;
```